bioimageio/core/build_spec/build_model.py

```diff
@@ -1,7 +1,7 @@
 import datetime
 import hashlib
 import os
-from pathlib import Path
+from pathlib import Path, WindowsPath
```

Comment:
WindowsPath is imported here but afaik not used in this file.

Reply:
right, found a nice way (problem was the potential ":" in windows paths interfering with splitting dependency manager and file at ":"). cleaned it up